Makefile

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+.PHONY: lint unit
+
+lint:
+	@echo -n '? '
+	docker-compose -f tools/golang/docker-compose.yml up lint
```

Comment:
If this is meant to be used in CI systems, you might want to provide at least the option of turning off all those muddying ANSI characters that get printed for things like e.g. coloring output. That's easily achieved by adding `--no-ansi` to the `docker-compose` commands, but the way Makefiles normally do stuff like that is by setting an opts variable that can be overridden on the command line. So something like:
```make
DC_OPTS ?= "--no-ansi"

.PHONY: lint

lint:
	@echo -n '? '
	docker-compose $(DC_OPTS) -f tools/golang/docker-compose.yml up lint

```
unless you think unprintable characters won't pose an issue? I think you're more familiar with Jenkins than I am.

Reply:
I dont think it would pose an issue but I dont see any issue or negative side affects with adding that option in! so added